template-vue-ts/tsconfig.json

```diff
@@ -0,0 +1,4 @@
+{
+  "files": [],
+  "references": [{ "path": "./tsconfig.app.json" }]
```

Comment:
I don't think it's necessary to use project references for a starter boilerplate... let's keep it simple with a single `tsconfig.json`.